src/team/crazynetwork/raids/Island.java

```diff
@@ -31,8 +33,7 @@ public Island(Player owner) {
         this.raidableTime = pIsland.getLong("raidableTime");
         for (String member : pIsland.getStringList("members"))
             this.members.add(Bukkit.getPlayer(UUID.fromString(member)));
-
-
+        this.balence = pIsland.getDouble("balence");
```

Comment:
Error checking to be added. Don't always expect config to be full of things. Include default values.